src/OpenRiaServices.DomainServices.Server/Framework/OpenRiaServices.DomainServices.Server.csproj

```diff
@@ -1,14 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFrameworks>net46;netcoreapp3.0</TargetFrameworks>
```

Comment:
Dont you need netstandard2.0 here aswell? or is it that net46 also supports netstandard2.0, so it's redundant? Since you are using it further down in the condition.